upstream-community-operators/federation/federation.v0.0.7.clusterserviceversion.yaml

```diff
@@ -3,14 +3,14 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
-  name: federation.v0.0.6
+  name: federation.v0.0.7
   namespace: placeholder
   annotations:
```

Comment:
Please add the `metadata.annotations.repository` to the CSVs, should have a link to the github repo for the operator